lib/widgets/gallery_view.dart

```diff
@@ -0,0 +1,258 @@
+import 'package:flutter/material.dart';
+import 'package:workoutpersonalizer_frontend/constants/styles.dart';
+
+class Gallery extends StatefulWidget {
+  final List<String> exerciseList;
+  final List<String> descriptionList;
+  final List<String> tagList;
+  final bool playlistGallery;
+
+  const Gallery(
+      {Key? key,
+      required this.exerciseList,
+      required this.playlistGallery,
+      required this.descriptionList,
+      required this.tagList})
+      : super(key: key);
+
+  @override
+  GalleryState createState() => GalleryState();
+}
+
+class GalleryState extends State<Gallery> {
```

Comment:
Could you make this class and its methods private (e.g. `_GalleryState`)
(except for the `build` method)

Reply:
:+1: